code/game/objects/items/draftingchalk.dm

```diff
@@ -18,37 +18,77 @@
 			smooth_icon(C)
 
 /obj/item/pen/drafting
-	name = "white drafting chalk"
-	desc = "A piece of white chalk for marking areas of floor."
+	name = "white chalk"
+	desc = "A piece of white chalk for marking areas of floor, or for drawing."
 	icon = 'icons/obj/crayons.dmi'
 	icon_state = "dchalk"
 	color = "#FFFFFF"
 	var/colorName = "whitec"
+	var/linemode = TRUE
+	var/instant = 0
 
 /obj/item/pen/drafting/attack_self(var/mob/user)
+	if(linemode)
+		linemode = FALSE
+		to_chat(user, SPAN_SUBTLE("You will now draw freestyle."))
+	else
+		linemode = TRUE
+		to_chat(user, SPAN_SUBTLE("You will now draw straight lines."))
 	return
 
-/obj/item/pen/drafting/afterattack(turf/target, mob/user, proximity)
-	if (!proximity || !istype(target, /turf/simulated) || target.density || target.is_hole)
-		return
-
-	to_chat(user, "You start marking a line on [target].")
+/obj/item/pen/drafting/examine(mob/user)
+	. = ..()
+	to_chat(user, SPAN_SUBTLE("Use the chalk on itself to change what you will draw."))
 
-	if (!do_after(user, 1 SECONDS, act_target = target))
+/obj/item/pen/drafting/afterattack(turf/target, mob/user, proximity)
+	if(linemode)
+		if (!proximity || !istype(target, /turf/simulated) || target.density || target.is_hole)
+			return
+
+		to_chat(user, "You start marking a line on [target].")
+
+		if(!do_after(user, 1 SECONDS, act_target = target))
+			return
+
+		for (var/obj/effect/decal/cleanable/draftingchalk/C in target)
+			qdel(C)
+
+		to_chat(user, "You mark a line on [target].")
+
+		var/obj/effect/decal/cleanable/draftingchalk/C = new(target)
+		C.color = color
+		target.add_fingerprint(user)
```

Comment:
not sure how drawing with chalk adds your fingerprints

Reply:
Once again, blame original crayon code.  I suppose because you have to kneel down and brace one hand against the floor while you draw with the other hand?  Let me know if you want me to fix it.

Reply:
ambivalent on it, your choice

Reply:
I left the fingerprints in.